<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>301</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lb_raceView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Рейсы</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_choiceAirp">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбор аэропорта</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="fr_radioBt">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="rb_out">
       <property name="text">
        <string>Вылет</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_in">
       <property name="text">
        <string>Прилёт</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pb_reciveRace">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Получить список рейсов</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_busyAirport">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Загруженность аэропорта</string>
    </property>
   </widget>
   <widget class="QFrame" name="fr_data">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>221</width>
      <height>43</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lb_data">
       <property name="text">
        <string>Дата</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>569</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
